### ISSUE TITLE
Consider adding to github/linguist

### ISSUE THREAD

**Issue by Purpzie** [rhaiscript/rhai#537]
I noticed that `.rhai` files don't have any highlighting on github, and that no issues exist about this yet. Github's highlighting is powered by [github/linguist](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-a-language), and they allow people to submit new languages if they're popular enough (used in >200 repos). Unfortunately github's search is pretty broken atm so I'm not sure how many repositories rhai is in.

Until github supports it, throwing this in a repo's `.gitattributes` will highlight it by pretending it's rust:
```gitattributes
*.rhai linguist-language=Rust
```

**Comment by schungx** [comment]
Rhai is a scripting language, so I'm not sure if it is actually used in any repo as a main development language.

However, I'll try the `.gitattributes` trick.

It might be better to use `js` in order to get support for interpolated strings...

**Comment by Purpzie** [comment]
True, but other scripting languages are already highlighted, such as lua.

Also oops yeah almost forgot about interpolated strings. I chose rust because I figured js would freak out without `()` in if statements, but it looks like the readme is handling `ts` just fine.

**Comment by schungx** [comment]
> Also oops yeah almost forgot about interpolated strings. I chose rust because I figured js would freak out without `()` in if statements.

It actually does OK with `if`...  I tried JavaScript on my own repo and it seems to work fine!

**Comment by Sable-20** [comment]
https://github.com/search?l=&q=a+extension%3Arhai&type=code

dunno if this is what youre looking for but here you go @Purpzie search for repos using rhai

**Comment by dankope** [comment]
Now that GitHub support cargo dependencies, there is clearly more than 200!
https://github.com/rhaiscript/rhai/network/dependents

**Comment by schungx** [comment]
I tried to submit to `github/linguist` but for some reason which I cannot remember I couldn't.

Maybe somebody can help do that?
